presto-docs/src/main/sphinx/spill/overview.rst

```diff
@@ -0,0 +1,74 @@
+.. contents::
+    :local:
+    :backlinks: none
+    :depth: 1
+
+Overview
+--------
+
+In the case of memory intensive operations, Presto features offloading
+intermediate operator results to disk. The goal of this mechanism is to
+enable execution of queries that require amounts of memory exceeding per query
+or per node limits.
+
+The mechanism is similar to OS level page swapping. However, it is
+implemented on the application level to address specific needs of Presto.
+
+Properties related to spilling are described in properties section
+:ref:`tuning-spilling`.
```

Comment:
Can you make sure this link works? is `tuning-spilling` already a reference in the properties section?